projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml

```diff
@@ -63,16 +63,15 @@ frontend-tests-vdk-jupyterlab-extension:
 
 
 release-vdk-jupyter:
+  extends: .common-setup-vdk-jupyterlab-extension
   variables:
     PLUGIN_NAME:
   image: "python:3.10-alpine"
   stage: release
   before_script:
     - export VDK_PATCH_VERSION=${CI_PIPELINE_ID}
```

Comment:
You need to move this to script section, you cannot have before_script twice- they do not get merged but get overriden.